### ISSUE TITLE
BLASR adds "RQ=" to read name when unmapped

### ISSUE THREAD

**Issue by nosarcasm** [mchaisso/blasr#3]
From BamUtils, I am seeing many INVALID_QNAMEs for unmapped reads (example below): 

>Number of records read = 534176
>Number of valid records = 508031

>Error Counts:
>       INVALID_QNAME: 26145

>TotalReads      534176.00
>MappedReads     508031.00
>PairedReads     0.00
>ProperPair      0.00
>DuplicateReads  0.00
>QCFailureReads  0.00
>
>MappingRate(%)  95.11
>PairedReads(%)  0.00
>ProperPair(%)   0.00
>DupRate(%)      0.00
>QCFailRate(%)   0.00
>
>TotalBases      4711273300.00
>BasesInMappedReads      4615088843.00
>Returning: 7 (INVALID)

When I look at mapped reads vs. unmapped, I am seeing this difference:

Mapped: 
> m150224_063755_42156_c100788292550000001823173308251556_s1_p0/133171/0_2111     0       007195F-001-01 ...

Unmapped: 
> m150225_013529_42156_c100779742550000001823166508251511_s1_p0/11992/37280_42077 RQ=0.787/0_4797 4       *

I think to fix this for the files I have right now is to convert back to SAM, strip out the RQ columns, and re-make the BAM

**Comment by mchaisso** [comment]
Are you using blasr from this repo or the pacbio one?

**Comment by nosarcasm** [comment]
This repo. Was this fixed in master?

**Comment by mchaisso** [comment]
No, but blasr does not attach the RQ field to the read name and does not
have a problem from bax.h5 files.  Are you aligning fasta files that have
this?  In any case, I pushed a fix for this issue.  Let me know if it works
for you.

-mark

**Comment by nosarcasm** [comment]
This was from fasta files, hot h5. Anyways, I'll try out the fix and let you know how it goes!
